bindings/python/pymongocrypt/version.py

```diff
@@ -12,4 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '1.0.2.dev0'
+try:
+    from pkg_resources import parse_version as _parse_version
+except ImportError:
+    from distutils.version import LooseVersion as _LooseVersion
+
+    def _parse_version(version):
+        return _LooseVersion(version)
```

Comment:
It's a little odd to put code in here since this is the single-source version file that's read in setup.py. Can we move these imports to binding.py?